README.md

```diff
@@ -49,6 +49,8 @@ When looking for support, please first search for your question in these venues:
 * [Node.js Help][]
 * [Open or closed issues in the Node.js GitHub organization](https://github.com/issues?utf8=%E2%9C%93&q=sort%3Aupdated-desc+org%3Anodejs+is%3Aissue)
 * [Questions tagged 'node.js' on StackOverflow][]
+* [Node.js Slack Community](http://node-js.slack.com): Visit 
+[nodeslackers.io](http://www.nodeslackers.io) to register 
```

Comment:
Micro-nit: period after `register`

Reply:
`http://www.nodeslackers.io` is not resolving for me, btw. Has it not yet been set up?